tests/test_sklearn_explain_prediction.py

```diff
@@ -303,3 +307,33 @@ def test_explain_tree_classifier_text(clf, newsgroups_train_big):
     res = explain_prediction(clf, docs[0], vec=vec, target_names=target_names)
     check_targets_scores(res)
     format_as_all(res, clf)
+
+
+def _assert_tree_explain_prediction_single_target(clf, X, feature_names):
```

Comment:
What do you think about moving this function to the top of the file, closer to other assert functions?